### PR TITLE
Paint 121

### PR DESCRIPTION
**Pull request by clempe**
Intro:
o Bugfixes
o Refactoring
o Added UI and Interrogation Test (Espresso)
o Extend EspressoHelper Class

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/listener/BottomBarScrollListener.java.orig [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
```
Comment (cbenkovic) [inline Paintroid/src/main/java/org/catrobat/paintroid/listener/BottomBarScrollListener.java.orig]: .../BottomBarScrollListener.java.orig ?!

**Comment by clempe** [comment]
?

Am 08.06.2017 13:57 schrieb "cbenkovic" <notifications@github.com>:

File: Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~HEAD [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/**
```
Comment (cbenkovic) [inline Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~HEAD]: ?

File: Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~HEAD_0 [hunk 3]
```diff
@@ -0,0 +1,86 @@
+/**
```
Comment (cbenkovic) [inline Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~HEAD_0]: ?

File: Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~bugfixes and added RTL featers [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/**
```
Comment (cbenkovic) [inline Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~bugfixes and added RTL featers]: ?

File: Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~bugfixes and added RTL featers_0 [hunk 5]
```diff
@@ -0,0 +1,71 @@
+/**
```
Comment (cbenkovic) [inline Paintroid/src/main/java/org/catrobat/paintroid/intro/helper/WelcomeActivityHelper.java~bugfixes and added RTL featers_0]: ?

**Pull request merged**